### DISCUSSION TITLE
Trantransaction got stuck

### DISCUSSION THREAD

**Discussion by getairdrop24** [zkSync-Community-Hub/zksync-developers#436] [General] [answered]
Mainnet

### Provide a brief description of the functionality you're trying to implement.

By the mistak i send my  0.044 ETH on mexc exchange on zksync era network and mexc exchange does not support of zkync era network in deposite.

It's taking too much time but nothing will happen. i already contact with mexc customer support but they said amonut not recived . so they are unable to do refund. 

please help

TxID=0xd88d6413d9f59ff2cbbc3a05f80b6bb6d3bd49f6c7ff42b7e3cd4861bc2d18a6
![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/85836509/f3426f70-48ad-4537-ae6f-96e7e32ba578)

### What is the specific issue or error you're encountering?

N/A

### Can you share the error messages or logs you're receiving, if any?

N/A

### Have you made any recent changes to the contract before encountering this issue?

N/A

### Are there any external libraries or contracts that your contract interacts with?

N/A

### Can you provide the relevant portions of your contract code where the issue is occurring?

N/A

### Have you tried to isolate the problem, and if so, what were the results?

N/A

### What steps have you already taken to try to resolve the issue?

N/A

**Comment by bxpana** [comment, marked as answer]
Thanks for reaching out, but you'll want to email us at support@zksync.io for proper support. This is for developer support questions.

**Reply by getairdrop24** [reply]
ok thanks
